catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/federation/base/AbstractFederationStrategy.java

```diff
@@ -49,6 +49,7 @@
   private static final String CLASS_NAME = AbstractFederationStrategy.class.getName();
 
   private static final int DEFAULT_MAX_START_INDEX = 50000;
+  private final Function<QueryRequest, QueryResponseImpl> queryResponseFactory;
```

Comment:
✏️ Personally, I would probably go with "converter" or "func" as my suffix. It doesn't really matter but "factory" implies something more to me.